src/projections/default-content.js

```diff
@@ -34,15 +34,21 @@ function getContent(td) {
   return td.content || defaultContent(td);
 }
 
-export default function ({
-  composeTds,
-}) {
-  return {
-    composeTds(td) {
-      return composeTds({
-        ...td,
-        content: getContent(td),
-      });
-    },
+export default function getDefaultContentProjection(factory) {
+  return function defaultContentProjection({
+    composeTds,
+    composeCaption,
+  }) {
+    return {
+      composeTds(td) {
+        return composeTds({
+          ...td,
+          content: getContent(td),
+        }).map(factory);
```

Comment:
this is incorrect, the factory should take the `content` not the `tdModel`.
```js
composeTds(td) {
  return composeTds({
    ...td,
    content: factory(getContent(td)),
  });
}
```

The same for the `composeCaption`.

Reply:
fixed